src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
```

Comment:
2. **Error 2: Unused private method**  
   The `unusedPrivateMethod` method is declared as private but never used. It should be removed or its usage should be justified.